Code/frame.js

```diff
@@ -991,6 +993,20 @@ define([
                         if(buttonData.ariaLabel) {
                             $button.attr('aria-label', buttonData.ariaLabel);
                         }
+                        //atribute to display tooltip
+                        $button.attr('data-toggle','tooltip')
+                        //to display tooltip with shortcut
+                        if(buttonData.key){
+                            $button.attr('data-html','true')
+                            $.each(buttonData.key,function(index,data) {
+                                var child = $('<div class="tooltip-shortcut-key">')
+                                child.text(data)
+                                $shortcutKeyContainer.append(child)
+                            })
+                            $tooltipConatiner.text(buttonData.tip)
```

Comment:
`tooltipConatiner` should be `tooltipContainer`

Reply:
This is not done.

Reply:
<img width="1407" alt="Screenshot 2022-09-22 at 2 26 24 PM" src="https://user-images.githubusercontent.com/111979571/191704631-3252ce6f-8444-44fa-bb6a-522569777bb9.png">

I have updated it in may latest commit, also can see it in files changed

Reply:
OK able to see now.